src/packages/recompose/__tests__/utils.js

```diff
@@ -7,8 +7,8 @@ import wrapDisplayName from '../wrapDisplayName'
 export const { act } = process.env.TEST_WITH_PREACT
```

Comment:
TODO:

- get `act` from `src/packages/recompose/__tests__/testing-library-setup.js` & remove conditional import from here
- should be able to remove `cross-env TEST_WITH_REACT_16=1` from `.github/workflows/test-react-16.yml`

Reply:
done in 5e5d7d3d33eeb51bcbecd6b2fe74a5042b1ca0e9